codehost/github/blame.go

```diff
@@ -201,3 +207,34 @@ func (c *GithubClient) SliceGitBlame(blame *GitBlame, filePath string, slices []
 
 	return slicedBlame, nil
 }
+
+func (c *GithubClient) ComputeGitBlameRank(gitBlame *GitBlame) []GitBlameAuthorRank {
+	var rankedAuthors []GitBlameAuthorRank
+	authorIndices := make(map[string]int)
```

Comment:
WDYT about using a map to gather authors' total lines and then convert them into an array and order it?

```suggestion
	totalLinesByAuthor := make(map[string]int)
```

Reply:
The thing is to do that we would need to have two loops first one to gather them and then another loop to convert them into an array, what do you think? should we separate the steps for more clarity